include-refactoring/xsimd/types/xsimd_all_registers.hpp

```diff
@@ -1,8 +1,12 @@
-#include "../types/xsimd_sse_register.hpp"
-#include "../types/xsimd_sse2_register.hpp"
-#include "../types/xsimd_sse3_register.hpp"
-#include "../types/xsimd_sse4_1_register.hpp"
-#include "../types/xsimd_sse4_2_register.hpp"
-#include "../types/xsimd_avx_register.hpp"
-#include "../types/xsimd_avx2_register.hpp"
-#include "../types/xsimd_avx512f_register.hpp"
+#include "xsimd_sse_register.hpp"
```

Comment:
or `'./'` to make it explicit it's a local header?

Reply:
In other projects of the xtensor-stack, we use `<>` for standard headers and "package/headers" for dependencies, so it's usually clear that an include with the file name only means it's a local heder. But both are fine for me.